nannou/src/app.rs

```diff
@@ -6,8 +6,6 @@
 //! - [**App**](./struct.App.html) - provides a context and API for windowing, devices, etc.
 //! - [**Proxy**](./struct.Proxy.html) - a handle to an **App** that may be used from a non-main
 //!   thread.
-//! - [**Draw**](./struct.Draw.html) - a simple API for drawing graphics, accessible via
```

Comment:
I wonder if we can keep this, but instead point to the `nannou::Draw` type (re-exported from the crate root)?

Some context in case you're interested - the `app` module used to have a thin wrapper `Draw` type that wrapped the "main" `draw::Draw` type. The idea was that it was a little simpler to use alongside the `app.draw()` method, but in the end we were able to simplify the main `draw::Draw` type enough that the `app::Draw` wrapper was no longer necessary. Looks like I forgot to update this path!

Reply:
I'm happy to make that change (I was originally considering doing just that), but I'm wondering if it might be confusing? This is the documentation for the `app` module after all, and if there isn't a `app::Draw` type or an `app::draw` module, it seems potentially confusing to mention `Draw` at the top here?

Are you thinking that users might start off reading the `app` module docs as an entry point to learn about the API surface, and so it would be good to link to `Draw` from here? Or is it more about backwards compatibility for users who are used to the original module layout?

Reply:
Taking a closer look I totally agree! At a glance, for some reason I thought this section was a reference to `Draw` in relation to the `app.draw()` method, but now I see this is the top-level docs for the module :blush: Yes removing it like this is certainly the right change!